test/lib/returnManageServer.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+const debug = require('debug')('manageServer');
+
+module.exports = returnManageServer;
+
+/**
+ * returnManageServer - Module to provide methods for before and after to start and close the server
```

Comment:
This is awesome :) are yall planning on using this to document your code thought the app !